DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -442,21 +461,27 @@ def verify_Syslog_daemon_forwarding_configuration(self):
             command_object = BasicCommand(command_name, command_to_run, result_keywords_array)
             command_object.run_full_test()
             if not command_object.is_successful:
-                command_object.print_error(self.Syslog_daemon_not_forwarding_error)
+                command_object.print_error(self.Syslog_daemon_not_forwarding_error(self.SYSLOG_DAEMON))
 
     def run_all_verifications(self):
-        '''
+        """
         This function is only called by main and runs all the tests in this class
-        '''
+        """
+        global FAILED_TESTS_COUNT
         if self.determine_Syslog_daemon():
             self.verify_Syslog_daemon_listening()
             self.verify_Syslog_daemon_forwarding_configuration()
+        else:
+            mock_command = BasicCommand(self.command_name, "")
```

Comment:
Why do you need the mock_command? Can't you just run:
self.print_result_to_prompts()
self.print_error(...)?